driver/src/main/java/org/neo4j/driver/internal/net/pooling/PoolSettings.java

```diff
@@ -53,4 +57,19 @@ public boolean idleTimeBeforeConnectionTestConfigured()
     {
         return idleTimeBeforeConnectionTest >= 0;
     }
+
+    public long maxConnectionLifetime()
+    {
+        if ( !maxConnectionLifetimeConfigured() )
+        {
+            throw new IllegalStateException(
+                    "Max connection lifetime is not configured: " + maxConnectionLifetime );
+        }
+        return maxConnectionLifetime;
+    }
+
+    public boolean maxConnectionLifetimeConfigured()
```

Comment:
Yeah, that's basically what it means in the code. Pool treats negative and zero as if connections have infinite lifetime. Do you think this method should reflect this in it's name?